.styleci.yml

```diff
@@ -0,0 +1,18 @@
+preset: symfony
+
+enabled:
+  - align_double_arrow
+  - newline_after_open_tag
+  - ordered_use
+  - long_array_syntax
+  - php_unit_construct
+  - php_unit_strict
+
+disabled:
+  - psr0
```

Comment:
It wouldn't be broken if you weren't using the .styleci.yml file, because we'd have updated the inbrowser config for you when we broke it.